special-interest-groups/sig-docs/membership.md

```diff
@@ -15,8 +15,12 @@
 
 ## Committers
 
+* [3pointer](https://github.com/3pointer)
+* [amyangfei](https://github.com/amyangfei)
+* [breeswish](https://github.com/breeswish)
```

Comment:
I agree with promoting breeswish & amyangfei to committers this time and expect other candidates to have a better command of both Chinese and English technical writing and documentation rules.

Reply:
Updated~